app/app.go

```diff
@@ -561,35 +559,43 @@ func NewStrideApp(
 	)
 	icaModule := ica.NewAppModule(&app.ICAControllerKeeper, &app.ICAHostKeeper)
 
-	// Create the middleware stacks
+	// Define the IBC stacks
+	// Stack one (ICAHost Stack) contains:
+	// - IBC
+	// - ICAHost
+	// - base app
+	icaHostIBCModule := icahost.NewIBCModule(app.ICAHostKeeper)
 
-	// Stack one contains
+	// Stack two (ICAController Stack) contains
 	// - IBC
-	// - ICA
-	// - icacallbacks
+	// - ICAController (middleware)
+	// - icacallbacks (middleware)
 	// - stakeibc
 	// - base app
-	var icamiddlewareStack porttypes.IBCModule
-	icamiddlewareStack = icacallbacksmodule.NewIBCModule(app.IcacallbacksKeeper, stakeibcIBCModule)
-	icamiddlewareStack = icacontroller.NewIBCModule(app.ICAControllerKeeper, icamiddlewareStack)
-	icaHostIBCModule := icahost.NewIBCModule(app.ICAHostKeeper)
+	icaCallbacksStack := icacallbacksmodule.NewIBCModule(app.IcacallbacksKeeper, stakeibcIBCModule)
+	icaControllerStack := icacontroller.NewIBCModule(app.ICAControllerKeeper, icaCallbacksStack)
```

Comment:
nit: is there a nice way to format this as a single line of code? something like
```
icamiddlewarestack := icacontroller.NewIBCModule(app.ICAControllerKeeper,
                                    icacallbacksmodule.NewIBCModule(app.IcacallbacksKeeper,
                                    stakeibcIBCModule)
```
this orders the stack in a visually intuitive way (not sure if the formatting works)

Reply:
I'm not sure there's a way to do that cleanly. Ideally, we'd flop the order of arguments if the NewIBCModule functions so that each one shows `NewIBCModule(sameStack, newKeeper)` but we can't change the ICA ones.

The main change here I was trying to get at was changing the naming of `icamiddlewarestack` since that doesn't differentiate between ICAHost or ICAController.

Depending on how we want to name the stack (see earlier comment about ICAController vs Stakeibc), we can do something similar to how you had it before with
```go
var icaControllerStack ibcporttypes.IBCModule = stakeibcIBCModule
icaControllerStack = icacallbacksmodule.NewIBCModule(app.IcacallbacksKeeper, icaControllerStack)
icaControllerStack = icacontroller.NewIBCModule(app.ICAControllerKeeper, icaControllerStack)
```